components/sidebar/ObserverSections.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+
+// import ObserverSections from './ObserverSections';
+import ObserverSections from './ObserverSections';
+
+describe('ObserverSections component', () => {
+    const dummyText =
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed pretium enim nec massa fringilla, ac ultrices tortor posuere. Fusce sed quam vitae arcu pharetra congue. Quisque in elementum nibh.';
+    const repeatText = (text, n) => {
+        return Array(n)
+            .fill(text)
+            .reduce((acc, cur) => cur + acc, '');
+    };
+    const list = [
+        { id: 'section1', granularity: 20, children: <div id="text-1">{repeatText(dummyText, 1)}</div> },
```

Comment:
Why do you need to set `id="text-1"`?

Reply:
It's not needed for anything. I will remove it.